### PR TITLE
Update workers.conf.j2

### PR DESCRIPTION
**Pull request by feanil**
Separate the app by the service variant so CMS and LMS report to separate apps.

Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?

### CODE REVIEW

**Review by robrap — COMMENTED** [review]
- add changelog entry
- add newrelic policies before merging

**Comment by feanil** [comment]
Alerting updates:
https://one.nr/0bRmdaZw
https://one.nr/08wpAq6R

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 Add any new changes to the top(right below this line).
 
+ - 2020-11-20
+     - Role: edxapp
+        - Updated the worker newrelic config to have the service variant in the app name.  This will seperate the names
+          of the newrelic apps to be `...-lms` and `...-cms` to make it easier to monitor them separately.  This will
+          impact an newrelic monitoring and alerting you have that is linked to the old app name.  It should be
+          duplicated to both of the new application names.
```
Comment (robrap) [inline CHANGELOG.md]:
```suggestion
          impact any newrelic monitoring and alerting you have that is linked to the old app name, which should be
          updated to use both of the new application names.
```

**Review by robrap — APPROVED** [review]
One suggested typo and clarity fix.

**Comment by robrap** [comment]
**Reminder:** We should update the alerting dashboards and documentation before announcing this change to Squads, and their new ability to put in place policies for workers.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 All notable changes to this project will be documented in this file.
 Add any new changes to the top(right below this line).
 
+ - 2020-11-20
+     - Role: edxapp
+        - Updated the worker newrelic config to have the service variant in the app name.  This will seperate the names
+          of the newrelic apps to be `...-lms` and `...-cms` to make it easier to monitor them separately.  This will
```
Comment (robrap) [inline CHANGELOG.md:8]: Nit: I imagine our recommendation for no hard line breaks in docs would apply to markdown as well. Also understood if you want consistency within a doc.

**Pull request merged**